apps/api/src/app/modules/tasks/tasks.service.ts

```diff
@@ -55,4 +63,13 @@ export class TasksService {
   async findOne(id: string): Promise<Task> {
     return this.taskModel.findById(id).exec();
   }
+
+  /**
+   * Cancel tasks
+   *
+   * @param ids Ids of the tasks
+   */
+  cancelMany(ids: string[]) {
+    return this.submitterService.CancelTasks({ task: { ids } });
```

Comment:
cela ne devrait il par être: 

`return this.submitterService.CancelTasks({ tasks: { ids } });`

(ajout du s à task)

Reply:
là, c'est pas moi qui décide les protos

https://github.com/aneoconsulting/ArmoniK.Api/blob/main/Protos/V1/submitter_service.proto#L86

Reply:
...